tests/ctst/steps/cronjobs/countItems.ts

```diff
@@ -2,12 +2,30 @@ import { When } from '@cucumber/cucumber';
 import Zenko from '../../world/Zenko';
 import { createJobAndWaitForCompletion } from 'steps/utils/kubernetes';
 
-When('the {string} cronjobs completes without error', { timeout: 1200000 }, async function (this: Zenko, job: string) {
+async function runCronjob(this: Zenko, job: string, site?: string) {
+    let zenkoName = 'end2end';
+
+    if (site === 'DR') {
+        zenkoName += '-pra';
+    }
+
     switch (job) {
     case 'CountItems':
-        await createJobAndWaitForCompletion(this, 'end2end-ops-count-items');
+        await createJobAndWaitForCompletion(this, `${zenkoName}-ops-count-items`);
+        break;
+    case 'EnsureVaultSeeds':
+        await createJobAndWaitForCompletion(this, `${zenkoName}-ops-vault-check-seeds`);
```

Comment:
instead of having a new list of jobs, may be better to just specify the actual name (or suffix) of the job in argument:

```suggestion
    await createJobAndWaitForCompletion(this, `${zenkoName}-ops-${job}`);
```

and call it accordingly: `when the count-items cronjobs completes without error`

(less code, less maintenance, less mistake because we give yet another name to the same thing. the benefit of such an "enum" would be to have compile-time validation, which is not the case here...)